tutorials/test_status/status_regex.yml

```diff
@@ -19,4 +19,26 @@ buildspecs:
     status:
       regex:
         stream: stdout
-        exp: "^(123FAIL)$"
\ No newline at end of file
+        exp: "^(123FAIL)$"
+
+  status_regex_stderr_pass:
+    executor: generic.local.bash
+    type: script
+    tags: [ system ]
+    description: Pass test based on regular expression
+    run: echo "PASS"
+    status:
+      regex:
+        stream: stderr
+        exp: "^(PASS)$"
+
+  status_regex_stderr_fail:
+    executor: generic.local.bash
+    type: script
+    tags: [ system ]
+    description: Pass test based on regular expression
+    run: echo "FAIL"
+    status:
+      regex:
+        stream: stderr
+        exp: "^(123FAIL)$"
```

Comment:
since you added 2 more tests here you will want to update the documentation in particular https://buildtest.readthedocs.io/en/devel/buildspecs/buildspec_overview.html#passing-test-based-on-regular-expression page where you should update the paragraph to explain what this test is doing. You should emphasize the important lines.